source/routing/journey.cpp

```diff
@@ -119,3 +119,15 @@ size_t JourneyHash::operator()(const Journey& j) const {
 
     return seed;
 };
+
+bool JourneyCmp::operator()(const Journey& l, const Journey& r) const {
```

Comment:
It would be nice to have some unit tests in `Journey_test.cpp` for this operator.